tests/integration/test_k8s.py

```diff
@@ -41,10 +40,10 @@ async def ready_nodes(k8s, expected_count):
         )
         for node in node_list["items"]
     }
-    log.info(f"Found {len(nodes)} nodes...")
+    log.info("Found %d/%d nodes...", len(nodes), expected_count)
```

Comment:
Why not an f-string ?

Reply:
log.info(f"Found {len(nodes)}/{expected_count} nodes...") ?

Reply:
Ahh b/c the logger library wants you to do it there way my friend.  Otherwise you get linter errors.  More specifically -- the logger library does lazy string interpolation so if you pass in some mega object that has a complex `__str__` method -- it doesn't get called if the log level isn't set to run it.  

Using an f-string would interpolate the str value of that object right away